### PR TITLE
[Fix] Show a message while user tries to create Material request with BOM

### PR DESCRIPTION
**Pull request by shreyasp**
- Validation message, when a user tries to create material request from BOM, which contains only one item which is non-stocked.

### CODE REVIEW

File: erpnext/stock/doctype/material_request/material_request.js [hunk 1]
```diff
@@ -127,14 +127,19 @@ erpnext.buying.MaterialRequestController = erpnext.buying.BuyingController.exten
...
-					$.each(r.message, function(i, item) {
-						var d = frappe.model.add_child(cur_frm.doc, "Material Request Item", "items");
-						d.item_code = item.item_code;
-						d.description = item.description;
-						d.warehouse = values.warehouse;
-						d.uom = item.stock_uom;
-						d.qty = item.qty;
-					});
+					if(!r.message){
+						frappe.throw(__("BOM does not contain any stocked items."))
```
Comment (anandpdoshi) [inline erpnext/stock/doctype/material_request/material_request.js]: should be `stocked item`. Also remove the dot at the end.

File: erpnext/stock/doctype/material_request/material_request.js [hunk 2]
```diff
@@ -127,14 +127,19 @@ erpnext.buying.MaterialRequestController = erpnext.buying.BuyingController.exten
...
+					}
+					else{
```
Comment (anandpdoshi) [inline erpnext/stock/doctype/material_request/material_request.js]: put else in the previous line and a space after `else`

**Comment by shreyasp** [comment]
@anandpdoshi Done the requested changes

**Comment by anandpdoshi** [comment]
merged manually after changing 'stocked' to 'stock'

**Pull request closed without merging**